src/types/ApiRequests.ts

```diff
@@ -19,7 +19,7 @@ export interface AuthRequest {
 }
 
 export interface EditProfileRequest {
-    photoUrl_base64: string | undefined;
+    photoUrlBase64: string | undefined;
```

Comment:
if you're going to be editing this, could you update your PR description so that others can look at the PR and know how to make the request? checkout Kate's recent PRs